script/testing/junit/traces/select.test

```diff
@@ -150,3 +150,28 @@ SELECT * FROM temp3 WHERE id=1 OR time=300 AND id=3 OR id=4 AND time=200 OR id=2
 
 statement ok
 DROP TABLE temp3
+
+# left outer join
+
+statement ok
+CREATE TABLE xxx (id INT PRIMARY KEY, val INT);
+
+statement ok
+CREATE TABLE yyy (id INT PRIMARY KEY, val INT);
+
+statement ok
+INSERT INTO xxx VALUES (2,2), (3,3), (4,4), (5,5);
+
+statement ok
+INSERT INTO yyy VALUES (2,2), (3,3), (4,4);
+
+query I nosort
+SELECT * FROM xxx LEFT OUTER JOIN yyy ON xxx.id = yyy.id;
```

Comment:
Are there more complex queries than this? For example, trying joining on the non-pkey columns.

And what about joining with an inequality:

`SELECT * FROM xxx LEFT OUTER JOIN yyy ON xxx.id > yyy.id + 1;`